tests/e2e/TimeoutConstants.ts

```diff
@@ -226,6 +226,6 @@ export const TimeoutConstants = {
     /**
      * Common timeout for plugins.
      */
-    TS_COMMON_PLUGIN_TEST_TIMEOUT: Number(process.env.TS_COMMON_PLUGIN_TEST_TIMEOUT) || 20_000
+    TS_COMMON_PLUGIN_TEST_TIMEOUT: Number(process.env.TS_COMMON_PLUGIN_TEST_TIMEOUT) || 20_000,
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.

Reply:
fixed with the linter code change